plugin.json

```
@@ -10,7 +10,10 @@
         "label": "Trigonometric transformation",
 
         // description: longer string to help end users understand what this plugin does
-        "description": "",
+        "description": "This plugin adds a processor in the prepare recipe : 'Custom processor trigonometric transformation'.\
```

summarize in a single line / 140 characters 
```suggestion
        "description": "Extract features from periodic columns (day of the week, latitude/longitude, etc.)"
```

---

Edit done.